demo/index.js

```diff
@@ -17,57 +21,143 @@ const getData = init({
 	config: '{}'
 }, embedContent);
 
-const volumeSlider = document.querySelector('#volumeSlider');
+addPlayerControls();
 
-const logEvents = ['error', 'load', 'volumeChanged', 'captionsChanged', 'playerStatusChanged', 'videoLoaded', 'seeked',
-	'webcastLoaded', 'webcastStarted', 'webcastEnded', 'broadcastStarted', 'broadcastStopped'];
-
-document.querySelector('#updateToken').addEventListener('click', () => {
-	const data = getData();
-	currentEmbed?.updateToken(data.config.token);
-});
+/**
+ * This demo page passes around data necessary for calling embedVideo or embedWebcast using this object interface
```

Comment:
This line should only be a brief description of the RevSDKDemoSettings type.
Put any long-form description in a block comment at the top

Reply:
it's jsdoc comments for Intellisense in IDEs. Will put at top of page